frontend/src/components/WindSettings.jsx

```diff
@@ -19,390 +19,153 @@ const useStyles = makeStyles((theme) => ({
+                    {renderSelectField("Wind Direction", windBlock.windDirection, (e) =>
+                        setWindBlockData(index, { windDirection: e.target.value }), WindDirection)}
+                    {renderTextField("Wind Velocity (m/s)", windBlock.windVelocity, (e) =>
+                        setWindBlockData(index, { windVelocity: e.target.value }), { min: 0 })}
+
+                    {(windBlock.windType === "Turbulent Wind") && (
+                        renderTextField("Fluctuation %", windBlock.fluctuationPercentage, (e) =>
+                            setWindBlockData(index, { fluctuationPercentage: e.target.value }), { min: 0, max: 100, step: 0.1 })
+                    )}
+
+                    <Grid item xs={12}>
+                        <IconButton onClick={() => deleteWindBlock(index)}>
+                            <DeleteOutline color="error" />
+                        </IconButton>
                     </Grid>
                 </Grid>
-                        
-                <Grid item container alignItems="center" direction="row">
-                    <Grid item xs={4}>
-                        <InputLabel id="fluctuation-label" sx={{ marginRight: 2, flexShrink: 0, color: '#F5F5DC', width: '200px' }}>
-                            Fluctuation %
-                        </InputLabel>
-                    </Grid>
-                    <Grid item xs={6}>
-                        <Tooltip title="Enter Fluctuation %" placement='bottom'>
-                            <TextField id="Fluctuation %" fullWidth
-                                variant="outlined" 
-                                type="number" 
-                                value={windShearData.fluctuationPercentage}
-                                sx = {{
-                                    backgroundColor: '#F5F5DC',
-                                    '& .MuiOutlinedInput-root': {
-                                        '& .MuiInputBase-input': {
-                                            padding: '6px 8px',
-                                        },
-                                    },
-                                }}
-                                onChange={(e) =>
-                                    setWindShearData({
-                                    ...windShearData,
-                                    fluctuationPercentage: e.target.value,
-                                    })
-                                }
-                                inputProps={{ min: 0, max: 100, step: 0.1 }} 
-                            />
-                        </Tooltip>
-                    </Grid>
+            ))}
+
+            <Grid item container xs={12}>
+                <Grid xs={10}
+                    classes={{ root: classes.backdropFilter }}
+                    sx={{ border: '1px white solid', textAlign: 'center' }}>
+                    <IconButton onClick={addNewWindBlock} color="warning">
+                        <AddIcon />
```

Comment:
@syedomair0 A small change here. Wrap the **`<AddIcon />` Add Wind Source** inside a button, so that the button label looks like **"+ Add Wind Source"**